content/source/docs/cloud/registry/using.html.md

```diff
@@ -105,35 +90,54 @@ module "vpc" {
+
+### Authentication
 
-Terraform Cloud can use your private modules during plans and applies with no extra configuration, _as long as the workspace is configured to use Terraform 0.11 or higher._
+To configure authentication to Terraform Cloud or your Terraform Enterprise instance, you can:
 
-A given Terraform Cloud workspace can only use private modules from the organization it belongs to. If you want to use the same module in multiple organizations, you should add it to both organizations' registries. (See [Sharing Modules Across Organizations](./publish.html#sharing-modules-across-organizations).) If you are using Terraform Enterprise v202012-1 or later, workspaces can also use private modules from organizations that are sharing modules with the workspace's organization.
+- (Terraform 0.12.21 or later) Use the [`terraform login`](/docs/cli/commands/login.html) command to obtain and save a user API token.
+- Create a [user API token][user-token] and [manually configure credentials in the CLI config file][cli-credentials].
+
+Make sure the hostname matches the hostname you use in module sources because if the same Terraform Cloud server is available at two hostnames, Terraform will not know that they are the same. To support multiple hostnames for module sources, use the `terraform login` command multiple times, and specify a different hostname each time.
+
+-> **Note** When SAML SSO is enabled, there is a [session timeout for user API tokens] (/docs/enterprise/saml/login.html#api-token-expiration), requiring you to periodically re-authenticate through the web UI. Expired tokens produce a _401 Unauthorized_ error.
 
-### On the Command Line
 
-If you're using Terraform 0.11 or higher, you can use private modules when applying configurations on the CLI. To do this, you must provide a valid [Terraform Cloud API token](../users-teams-organizations/users.html#api-tokens).
 
 #### Permissions
 
-When you authenticate with a user token, you can access modules from any organization you are a member of. (A user is a member of an organization if they belong to any team in that organization.) If you are using Terraform Enterprise v202012-1 or higher, you can also access modules from any organization that is sharing modules with any organization you are a member of.
+You can use either a [user token](/docs/cloud/users-teams-organizations/users.html#api-tokens) or a [team token](/docs/cloud/users-teams-organizations/api-tokens.html#team-api-tokens) for authentication, but the type of token you choose may grant different permissions.
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+- **User Token**: Allows you to access modules from any organization in which you are a member. You are a member of an organization if you belong to any team in that organization. In Terraform Enterprise v202012-1 or higher, you can also access modules from any organization that is sharing modules with any of your organizations.
```

Comment:
@hashitop Is this something that we should also add below for the team token as well? I'm not 100% sure where this applies and where it doesn't apply.

Reply:
The application supports all these types of tokens but I do understand that we are trying to focus on the use cases which I may not be able to cover every single one. If we have those supported token types listed with the recommended scenario that would be really helpful. Perhaps instead of the **Note** that is highlighted in the **Authentication** section, it can be moved into this section instead. Additionally, this can be highlighted together with an option to use the [IsServiceAccount](https://www.terraform.io/docs/enterprise/saml/attributes.html#isserviceaccount) SAML attribute to avoid session timeout.

```suggestion
- **User Token**: Allows you to access modules from any organization in which you are a member. You are a member of an organization if you belong to any team in that organization. In Terraform Enterprise v202012-1 or higher, you can also access modules from any organization that is sharing modules with any of your organizations.

-> **Note** When SAML SSO is enabled, there is a [session timeout for user API tokens] (/docs/enterprise/saml/login.html#api-token-expiration), requiring you to periodically re-authenticate through the web UI. Expired tokens produce a _401 Unauthorized_ error. A SAML SSO account with [IsServiceAccount](https://www.terraform.io/docs/enterprise/saml/attributes.html#isserviceaccount) is treated as a service account and will not have the session timeout.
```